great_expectations/expectations/metrics/query_metrics/query_column_pair.py

```diff
@@ -36,7 +36,7 @@ def _sqlalchemy(
         metric_value_kwargs: dict,
```

Comment:
This file had 9 typing errors

https://github.com/great-expectations/great_expectations/blob/develop/pyproject.toml#L99

Reply:
Taken care of.